warehouse/malware/tasks.py

```diff
@@ -0,0 +1,26 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import warehouse.malware.checks as checks
+
+from warehouse.tasks import task
+
+
+@task(bind=True, ignore_result=True, acks_late=True)
+def run_check(task, request, check_name, check_id, file_id):
+    try:
+        check = getattr(checks, check_name)(check_id)
```

Comment:
Just to make sure I understand: `check_name` here would be something like `ExampleCheck` (i.e. the class name), right?

So part of the development process for new checks includes explicitly importing them into the `checks` namespace (which SGTM, just want to make sure I'm on the same page 😄)

Reply:
Yes, that's what I'm thinking now. Open to other suggestions though!

Reply:
That sounds good to me! I've seen and done hacky things with dynamic loading of classes into a fake namespace before and that's consistently been a mistake, so leveraging the actual Python module model sounds good.